src/Website/google classroom.json

```
@@ -0,0 +1,46 @@
+{
+  "google classroom.home": {
+    "message": "Homepage",
+    "description": "Displayed when the user is on the home page"
+  },
+  "google classroom.calendar": {
+    "message": "Viewing the calendar",
+    "description": "Displayed when the user viewing their calendar"
+  },
+  "google classroom.todo": {
+    "message": "Viewing to-do list",
+    "description": "Displayed when the user is viewing their to-do list"
+  },
+  "google classroom.assignmentPrivate": {
+    "message": "Viewing an assignment",
+    "description": "Displayed when the user is viewing an assignment and their privacy mode is on"
```

Explain what "privacy mode is on" means

---

Privacy mode is a setting within the presence which disables other users from being able to see the class name etc
Look at the [PR](https://github.com/PreMiD/Presences/pull/3203) to see what it does, in terms of screenshots

---

Yeah I know but it may be harder for translations to understand so I think you should give a slightly more detailed explanation.
```suggestion
    "description": "Displayed when the user is viewing an assignment and they chose to share the name of the assignment"
```
Doesn't need to be this because I'm not sure if that's fully correct but something along these lines

---

Would this be much better?

Displayed when the user is viewing an assignment and their privacy mode is on (privacy mode is a setting within the presence to hide sensitive, in this case, assignment)

---

Yep, sounds good. You can use that to all strings that use privacy mode